test/test_requirements.py

```diff
@@ -0,0 +1,127 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import csv
+import os
+import re
+import subprocess
+from importlib.metadata import version
+
+REQ_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.txt")
+)
+REQ_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "requirements.csv")
+)
+DOC_TXT = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.txt")
+)
+DOC_CSV = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), "..", "doc", "requirements.csv")
+)
+
+CACHE_PATH = os.path.join(os.path.expanduser("~"), ".cache", "cve-bin-tool")
+
+CACHE_CSV = os.path.join(CACHE_PATH, "requirements.csv")
+
+HTML_DEP_PATH = os.path.abspath(
+    os.path.join(
+        os.path.dirname(__file__),
+        "..",
+        "cve_bin_tool",
+        "output_engine",
+        "html_reports",
+        "js",
+    )
+)
+HTML_DEP_CSV = os.path.join(HTML_DEP_PATH, "dependencies.csv")
+
+# Dependencies that currently have CVEs
+# Remove from the list once they are updated
+ALLOWED_PACKAGES = ["reportlab"]
+
+
+def get_out_of_sync_packages(csvfile, txtfile):
+
+    out_of_sync_packages = []
+    csv_package_names = []
+
+    with open(csvfile, "r") as csvf, open(txtfile, "r") as txt:
+        csvfl = csv.reader(csvf)
+        next(csvfl)
+        for (vendor, product) in csvfl:
+            csv_package_names.append(product)
+        lines = txt.readlines()
+        for line in lines:
+            txt_package = re.split(">|\\[|;|=|\n", line)[0]
+            if txt_package not in csv_package_names:
+                out_of_sync_packages.append(txt_package)
+
+    return out_of_sync_packages
```

Comment:
You only check if new dependencies were added in .txt, but what if some were deleted?
Basically you return `set(txt_deps) - set(csv_deps)` but you should also take into account `set(csv_deps) - set(txt_deps)`.
Combining them together might be confusing, so let's use namedtuple maybe? Then you'll need to change logic in the `test_txt_csv_sync` to distinguish between them too.

Reply:
Even thought the chance for that to happen can be less likely it's better to test that too. Thanks!

Reply:
I think you misunderstood me. I wasn't implying that someone would add new packages to the .csv and forget .txt, I meant when cve-bin-tool drops some requirements (this can happen) so they are being deleted from .txt but still are in .csv. In this case you should say "Delete extra stuff from .csv, please" because we treat .txt as the source of truth. Now you say "Please add stuff to .txt".

So variable `out_of_sync_txt_packages` should be `new_packages` and `out_of_sync_csv_packages` should be `removed_packages` imo. With appropriate changes further down.